BookingsApi/BookingsApi/Controllers/JudiciaryPersonController.cs

```diff
@@ -90,7 +90,7 @@ public async Task<IActionResult> BulkJudiciaryPersonsAsync(IEnumerable<Judiciary
                 }
                 catch (Exception ex)
                 {
-                    _logger.LogError(ex, bulkItemErrorMessage, item.Id);
+                    _logger.LogError(ex, bulkItemErrorMessage, item.PersonalCode);
```

Comment:
when logging it's best to format like below to maintain the logging structure when captured by whatever logging tool/sink you use. It's a very good and recommend practice we should look out for 
`_logger.LogError(ex, "Could not add or update external Judiciary user with Personal Code: {PersonalCode}", item.PersonalCode);`

Reply:
fair enough I think you could probably create a reusable string with this alone still `"Could not add or update external Judiciary user with Personal Code: {PersonalCode}"` and it should still allow you maintain a reusable message and the logging structure